documentation-samples/quickstarts/Face/FaceQuickstart.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Are you sure you want to check-in sln and csproj files?

Reply:
I deleted them, thanks.